### PR TITLE
Edit Getting Started with JS and Options API

### PR DESCRIPTION
**Pull request by yordan-mitev**

### CODE REVIEW

File: docs/getting-started/javascript-options-api.md [hunk 1]
```diff
@@ -104,22 +101,25 @@ Kendo UI for Vue includes [four artfully designed themes](slug:themesandstyles)
...
  
-You can add any additional custom styles in the `<styles>` tag of the `src/App.vue` file.
+To add any custom styles to your app, insert a `<styles>` tag in the `src/App.vue` file and place your styles there.
  
 ## Add a Vue Data Grid Component
  
 Now that you've installed all required packages, you are ready to add the Kendo UI for Vue Data Grid to the application:
  
-1. In the `src/App.vue` file, add a `<script>` block and import the Grid and its data:
+1. In the `<script>` block of the `src/App.vue` file, import the Grid and its data. In addition, the `process` function from the [Data Query](https://www.telerik.com/kendo-vue-ui/components/dataquery/) package will allow you to use data operations like sorting, paging, and filtering.
```
Comment (filipKovachev) [inline docs/getting-started/javascript-options-api.md:115]: I think it is better to say  will allow you to apply data operations like sorting, paging, and filtering`

File: docs/getting-started/javascript-options-api.md [hunk 2]
```diff
@@ -156,129 +159,158 @@ Now that you've installed all required packages, you are ready to add the Kendo
...
-    - Set the initial `skip` for the paging.
-    - Set the initial sorting by Product name.
+
     - Set `sortable` to `true`.
     - Set `pageable` to `true`.
-    - Initialize the `dataResult` empty array.
+    - Set the page size (`take`) to `10` to define how many items are rendered on each page.
+    - Set the initial `skip` value, which allows the Grid to calculate how many items to skip when paging.
+    - Set the initial sorting by product name.
+    - Initialize the `dataResult` empty array, which returns the results from the sorting and paging.
```
Comment (filipKovachev) [inline docs/getting-started/javascript-options-api.md:221]: the `dataResult` doesn't really return the results from sorting and paging, it rather holds them.

File: docs/getting-started/javascript-options-api.md (hunk 1 above)
Comment (yordan-mitev) [inline docs/getting-started/javascript-options-api.md:115]:
```suggestion
1. In the `<script>` block of the `src/App.vue` file, import the Grid and its data. In addition, the `process` function from the [Data Query](https://www.telerik.com/kendo-vue-ui/components/dataquery/) package will allow you to apply data operations like sorting, paging, and filtering.
```

File: docs/getting-started/javascript-options-api.md (hunk 2 above)
Comment (yordan-mitev) [inline docs/getting-started/javascript-options-api.md:221]:
```suggestion
    - Initialize the `dataResult` empty array, which holds the results from the sorting and paging.
```

**Review by filipKovachev — APPROVED** [review]

**Pull request merged**